yarn.lock

```diff
@@ -0,0 +1,3846 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I'm a bit weary to re-introduce this, as we originally removed the lockfile in #9. But now that I read this article, published after that PR (https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all) I feel a little better about the possibility of including it. 

I wonder if it would be worthwhile to possibly add this to the `.npmignore` to remove the bulk of almost 3k lines to the package size. I'll do some research on it.